lua/astrocommunity/motion/leap-nvim/init.lua

```diff
@@ -5,26 +5,28 @@ return {
     {
       "AstroNvim/astrocore",
       opts = {
-        autocmds = {
-          leap_cursor = { -- https://github.com/ggandor/leap.nvim/issues/70#issuecomment-1521177534
-            {
-              event = "User",
-              pattern = "LeapEnter",
-              callback = function()
-                vim.cmd.hi("Cursor", "blend=100")
-                vim.opt.guicursor:append { "a:Cursor/lCursor" }
-              end,
-            },
-            {
-              event = "User",
-              pattern = "LeapLeave",
-              callback = function()
-                vim.cmd.hi("Cursor", "blend=0")
-                vim.opt.guicursor:remove { "a:Cursor/lCursor" }
-              end,
-            },
-          },
-        },
+        autocmds = vim.version().minor < 10
+            and {
+              leap_cursor = { -- https://github.com/ggandor/leap.nvim/issues/70#issuecomment-1521177534
+                {
+                  event = "User",
+                  pattern = "LeapEnter",
+                  callback = function()
+                    vim.cmd.hi("Cursor", "blend=100")
+                    vim.opt.guicursor:append { "a:Cursor/lCursor" }
+                  end,
+                },
+                {
+                  event = "User",
+                  pattern = "LeapLeave",
+                  callback = function()
+                    vim.cmd.hi("Cursor", "blend=0")
+                    vim.opt.guicursor:remove { "a:Cursor/lCursor" }
+                  end,
+                },
+              },
+            }
+          or {},
```

Comment:
Is this vim.version() api available on 0.9?

Reply:
Was added in 0.9 by the looks of it https://neovim.io/doc/user/news-0.9.html

> Added [vim.version](https://neovim.io/doc/user/lua.html#vim.version) for parsing and comparing version strings conforming to the semver specification.